src/error.ts

```diff
@@ -409,7 +409,7 @@ export class MongoCursorInUseError extends MongoAPIError {
  * @category Error
  */
 export class MongoServerClosedError extends MongoAPIError {
-  constructor(message: string) {
+  constructor(message = 'Server has been closed') {
```

Comment:
Are we all in agreement on tense? super bikeshedding but present/imperative seems more ideal

Reply:
@dariakp @emadum @W-A-James? Personally, I'm in favor of present tense but happy to defer.

Reply:
My vote would also be for using present/imperative.